src/build-remote/build-remote.cc

```diff
@@ -64,6 +65,8 @@ int main (int argc, char * * argv)
 
         settings.maxBuildJobs.set("1"); // hack to make tests with local?root= work
 
+        initPlugins();
```

Comment:
Why are all these `initPlugins()` calls necessary? Can't that be done in `initNix()` or some other common location?

Reply:
They need to be called after `--option` flags are parsed, which is done in a bunch of different ways by different programs.